src/chief.t.sol

```diff
@@ -247,23 +283,54 @@ contract DSChiefTest is DSThing, DSTest {
         uLarge.doVote(uLargeSlate);
     }
 
-    function testFail_lift_while_out_of_order() public {
-        initial_vote();
+    // TODO: Review the sense of this test as it was failing due of lack of approval and not the last lift
 
-        // Upset the order.
-        uSmall.doApprove(gov, address(chief), uSmallInitialBalance);
-        uSmall.doLock(uSmallInitialBalance);
+    // function testFail_lift_while_out_of_order() public {
+    //     initial_vote();
 
-        address[] memory uSmallSlate = new address[](1);
-        uSmallSlate[0] = c3;
-        uSmall.doVote(uSmallSlate);
+    //     // Upset the order.
+    //     uSmall.doApprove(gov, address(chief), uSmallInitialBalance);
+    //     uSmall.doLock(uSmallInitialBalance);
 
-        uMedium.doFree(uMediumInitialBalance);
+    //     address[] memory uSmallSlate = new address[](1);
+    //     uSmallSlate[0] = c3;
+    //     uSmall.doVote(uSmallSlate);
 
-        chief.lift(c3);
+    //     hevm.roll(2);
+    //     uMedium.doApprove(iou, address(chief), uMediumInitialBalance);
+    //     uMedium.doFree(uMediumInitialBalance);
+
+    //     chief.lift(c3);
+    // }
```

Comment:
I would fix the approval and make sure the fail happens on the out-of-order `lift()`.

Reply:
That's actually the problem, I'm not really sure what this test wants to prove. If the approval is added, the test doesn't have any revert. I couldn't get the sense of it.

Reply:
Checking it in master doesn't make any sense to me. Not really sure what "lift_while_out_of_order" wants to mean.

Reply:
Okay, yes, I also don't see why this would fail.

Reply:
So, this test has had essentially the same form since it was first added:
https://github.com/dapphub/ds-chief/commit/e034e2b20232bb3f928eeb31328d497c82ea1db1

Here's my conjecture as to what the intention was:
The vote by the small user makes `c3` the leader in votes.
The current `hat` is `c1`, so if at this point `c3` could be `lifted`.
The `free` by the medium user removes votes from `c3`, and the hope was that this renders `c3` not the leader in votes any longer so the `lift` fails.
Unfortunately, the medium user was the only one that voted on any candidate, so `c3` is still actually the leader after the `free`.

So maybe the sense of the test is captured if instead of freeing, the medium user just re-votes on `c1` and `c2` so that `c3` cannot be lifted?

Reply:
IMO it is also fine to just remove what was clearly an incorrectly-implemented test.

Reply:
Yeah let's remove it.